.travis.yml

```diff
@@ -1,12 +1,36 @@
+# Register this repo at https://whitewater.ibm.com/tools/travis
+
+# We don't know and don't care about the starter language, but use npm and node inside some scripts:
 language: node_js
+
+# Specify a level for stability.
 node_js:
-  - "6"
-  - "7"
-  - "8"
+  - '5.5.0'
+
 cache:
   directories:
     - "node_modules"
 
+# Nothing special to install since we're just verifying compatibility of this repo with the
+# developer experience infrastructure.
+install: true
+
+# Clone the repository that has the smart script for validating this repo against the registry
+# microservice (which itself validates against scaffolder, therefore Yeoman and SDKGen microservices).
+# It's the kitchen sink of validation!
+before_script:
+  - git clone -b v1 git@github.ibm.com:bluemix-mobile-services/bmd-devops-pipeline.git
```

Comment:
can you add `bmd-devops-pipeline` to the cache modules along with `node_modules`?